Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM centos:centos7
+
+EXPOSE 8080
+
+ADD sendEmail-epel-7.repo /etc/yum.repos.d/
+
+RUN yum install -y epel-release && \
+    yum -y --setopt=tsflags=nodocs update && \
+    yum -y --setopt=tsflags=nodocs install httpd \
+                                           nagios \
+                                           telnet \
+                                           supervisor \
+                                           python-jinja2 \
+                                           nagios-plugins-all \
+                                           sendEmail \
+                                           perl-Net-SSLeay \
+                                           perl-IO-Socket-SSL && \
+    yum clean all && \
+    mkdir -p /opt/rhmap/ && \
+    sed -i -e 's/Listen 80/Listen 8080/' /etc/httpd/conf/httpd.conf && \
+    sed -i -e 's|DocumentRoot "/var/www/html"|DocumentRoot "/usr/share/nagios/html"|' -e 's|<Directory "/var/www">|<Directory "/usr/share/nagios/html">|' /etc/httpd/conf/httpd.conf && \
+    touch /supervisord.log supervisord.pid && \
```

Comment:
Maybe it should be absolute path, `/supervisord.pid`?
I see the absolute path is referenced later on.

Since we don't set the WORKDIR, I'd avoid using relative paths.